docs/index.html

```diff
@@ -354,26 +355,26 @@ <h2 id="blur">Gaussian Blur</h2>
             In theory, the Gaussian function is infinite.
             It means that we need to compute every pixel of the source image
             for every pixel in the destination image.
-            In practice, Gaussian is rapidly decreasing function
-            and window size more than 3 * sigma has no meaning.
+            In practice, Gaussian is a rapidly decreasing function
+            and a window size more than 3 * sigma has no meaning.
         <p>
             As a result, Gaussian blur performance should depend
-            on windows size and sigma.
+            on window size and sigma.
             For most implementations this is true.
             But in 2011 Mathematical Image Analysis Group
             from Saarland University
             <a href="http://www.mia.uni-saarland.de/Publications/gwosdek-ssvm11.pdf">
                 proved</a>
             that Gaussian blur can be very closely approximated by
             series of extended box filters.
-            In opposite to true Gaussian filter,
+            Unlike a true Gaussian filter,
             box filters can be performed in constant time
             relative to blur radius.
         <p>
             This approach was implemented in Pillow <code>2.7.0</code>.
             According to the tests, even if other convolution implementations
-            can beat box filters approximation on small sigma,
-            this benefit disappears alongside with sigma increasing.
+            are superior to box filters' approximation with small sigma,
```

Comment:
I believe it easy to read "approximation with small sigma" together, while "with small sigma" is about operation, not approximation. How about this:

are superior to box filters' approximation on operations with small sigma,